src/LSPEncoder.cpp

```diff
@@ -2,7 +2,7 @@
 
 	LSP (Light Speed Player) Converter
 	Fastest & Tiniest 68k MOD player ever!
-	Written by Arnaud Carr� aka Leonard/Oxygene (@leonard_coder)
+	Written by Arnaud Carré aka Leonard/Oxygene (@leonard_coder)
```

Comment:
seems your editor xform original ANSI text file into UTF8. Like, "é" is now 2 bytes in the text file, should be ANSI single byte

Reply:
Damn, I should have just left those as is.  I assumed it was broken and that it might annoy you to have your name borked.  I don't find an "ANSI" encoding option in CLion though, a bunch of Windows ones, I will try to figure out which one keeps it correct.

Reply:
OK I think this is fixed now, still shows Carré but with ISO-8859-1 encoding now.